From 5e41f3b5f40f3350b956acbedfdb6b60464a5d80 Mon Sep 17 00:00:00 2001
From: Adrian Fiergolski <adrian.fiergolski@cern.ch>
Date: Mon, 30 Jan 2017 13:48:27 +0100
Subject: [PATCH] Remove CONFIG_SPL_OS_BOOT.

It has a problem to load uImage, thus Caribou loads OS using u-boot.
In newer version of Yocto it will by KCONFIG paramter, thus this patch will need to be updated.

Signed-off-by: Adrian Fiergolski <adrian.fiergolski@cern.ch>
---
 include/configs/zynq-common.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/include/configs/zynq-common.h b/include/configs/zynq-common.h
index c2a58cb..38d26b0 100644
--- a/include/configs/zynq-common.h
+++ b/include/configs/zynq-common.h
@@ -467,9 +467,6 @@
 #define CONFIG_SPL_ETH_DEVICE "Gem.e000b000"
 #endif
 
-/* for booting directly linux */
-#define CONFIG_SPL_OS_BOOT
-
 /* SP location before relocation, must use scratch RAM */
 #define CONFIG_SPL_TEXT_BASE	0x0
 
-- 
2.9.3

